src/runtime_src/core/tools/xbmgmt2/CMakeLists.txt

```diff
@@ -1,4 +1,15 @@
 # -----------------------------------------------------------------------------
+if(NOT WIN32)
+if ((${LINUX_FLAVOR} STREQUAL SUSE))
+set(Boost_USE_STATIC_LIBS OFF)               # find dynamic libraries
+else()
+set(Boost_USE_STATIC_LIBS ON)               # Only find static libraries
+endif()
+else()
+set(Boost_USE_STATIC_LIBS ON)               # Only find static libraries
+endif()
+set(Boost_USE_MULTITHREADED ON)             # Multi-threaded libraries
+
```

Comment:
Boost configuration must no be controlled in CMakeLists.txt post 2020.2.  If you notice this has changed in master as compared to 2020.2, we link dynamically with Boost for everything now.  Please remove.

Reply:
done